postem/bin/.gitignore

```diff
@@ -0,0 +1 @@
+/org/
```

Comment:
Remove this file, we have a general gitignore.

Reply:
Thank you, Miguel.
Done.